### ISSUE TITLE
problem when press login by sms or email 

### ISSUE THREAD

**Issue by ahmedemad2345** [iamsayan/fb-account-kit-login#2]
hello i have installed this plugin in my site and after set all require data and save it 
i have copy the short code to my login page after that i go to test it and the button to login is appear but when i click to login by any button the screen shoot appear 
![2019-06-20_041508](https://user-images.githubusercontent.com/43632744/59813515-1440db80-9312-11e9-9079-2bcb9ff8856a.png)

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

Do you have configured the Facebook App Settings properly?

**Comment by ahmedemad2345** [comment]
yes, it's working now but the real problem is when i send sms and verified it the page redirect me to a WordPress login page and say you are not register in this site

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

Please enable New user Registration from plugin settings and save settings.

![image](https://user-images.githubusercontent.com/25281493/59830037-69a1da80-935c-11e9-82ab-ec2159f011e0.png)

Hope it will work. Thanks!

**Comment by ahmedemad2345** [comment]
sorry for delay , yes it's active but when i ask for the code and enter it redirect me to wordpress login page and i want to redirect me to the my account page to complete registration

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

Please update this plugin to latest version i.i. v1.2.0 and check it again. If the problem still exists, please let me know.

Thanks!

**Comment by ahmedemad2345** [comment]
hey bro thank you for update plugin but the problem still when i register using sms the verify code sent successfully but when i put it to sign up this message appear Please wait until we authenticate you.
 and no thing happen although i have edit the setting of plugin to redirect to my home page

**Comment by iamsayan** [comment]
Hi, I am very sorry for the problem you have must caused. What is your website? Could you please share yout login (username and password) details to this email address: hello@sayandatta.com. 

It will help me to investigate and fix this issue by logging in to your website. Do not worry, your sensitive info will be safe.

Thaanks!

**Comment by ahmedemad2345** [comment]
ok thank you i will send it to you

**Comment by ahmedemad2345** [comment]
hey bro i sent it to you when you will start check

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

It seems that any of the plugin blocks the ajax request on your website and you do not update your plugin regularly. Please update all plugins andlet me know.

Thanks!

**Comment by ahmedemad2345** [comment]
ok thank you bro for your time i will do that and tell you as soon as i finished

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

Please update this plugin to v1.2.1 and see. I think it will solve your problem.

Thanks!

**Comment by ahmedemad2345** [comment]
ok thank you bro i will do and tell you the results

**Comment by ahmedemad2345** [comment]
hey bro it's working very good thank you very much you are a good friend

**Comment by ahmedemad2345** [comment]
hey bro i have update the plugin and it's not redirect to home page after register can you please help me to fix it

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

Then where it will redirect you?

**Comment by ahmedemad2345** [comment]
it's not redirect any way bro you can try it[ here  ](x-vendor.com) if you want to be an admin just ask to send email to you to login to dashboard

**Comment by ahmedemad2345** [comment]
https://x-vendor.com/

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

Please try to change these to your desired page URL from plugin settings.

![image](https://user-images.githubusercontent.com/25281493/62799201-d2f0cf00-bafd-11e9-8c52-624ea23bbedd.png)

Hope it helps. If it will not work, please give me the login credintials via email to access admin account to accress plugin settings. Thanks!

**Comment by ahmedemad2345** [comment]
ok bro i tried to edit it but it doesn't work so i send you an account to test it yourself

**Comment by iamsayan** [comment]
hi @ahmedemad2345 

Where did you send login details?

**Comment by ahmedemad2345** [comment]
hi bro i send it to this e-mail hello@sayandatta.com

**Comment by ahmedemad2345** [comment]
hey bro any news

**Comment by iamsayan** [comment]
Hi @ahmedemad2345 

Sorry for the late reply actully i was so busy for past few days. I have checked my email, but your email was not found there. So, please send me once again at hello@sayandatta.com.

Thanks!

**Comment by ahmedemad2345** [comment]
ok bro i have sent it again thank you

**Comment by ahmedemad2345** [comment]
hello , bro i didn't here from you for a long time are you ok ? 
if this version of plugin has a problem please send me the version before this it was work correctly thank you

**Comment by iamsayan** [comment]
Hi,

Here is the file: 

[fb-account-kit-login.1.2.2.zip](https://github.com/iamsayan/fb-account-kit-login/files/3532846/fb-account-kit-login.1.2.2.zip)

Thanks!

**Comment by ahmedemad2345** [comment]
no problem bro i will check it and replay you thank you very much
